examples/specs/errorband_ci_line.vl.json

```diff
@@ -1,37 +1,21 @@
 {
```

Comment:
Can we name the file `layer_line_errorband_ci`? (same for others since it's more like layering line with errorband_ci than the other way round.)

Reply:
Should I name the scatter plot graph with global mean and standard deviation `scatter` or `point`?